package exception;

public class ExceptionDemo {

	public static void main(String[] args) {
		int a = 5;
		int b = 0;
		int c = 0;
		
		System.out.println("Going to divide...");
		try {
			c = a / b;
		}
		
		System.out.println("Division over...");
		System.out.println("Answer is : " + c);
	}

}
